.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Minor: why not `3.8` since it is the latest?

Reply:
I was like this on Travis, but I agree.

Reply:
Good catch. Fixed it